src/components/DeckMechanisms/index.js

```diff
@@ -125,14 +129,29 @@ export default class DeckMechanisms extends React.Component {
         // Remove the played card from the hand.
         newState.hand = state.hand.filter(cardId => cardId !== id)
 
-        // If the card played is a Frozen Core, increment the amount of active
-        // Frozen Cores by 1.
-        if (id === 'W9') {
-          newState.specifics.activeFrozenCores += 1
-        }
-
-        if (id === 'W16') {
-          newState.specifics.liveDawnsparks = true
+        switch (id) {
+          case 'W9':
+            // If the card played is a Frozen Core, increment the amount of active
+            // Frozen Cores by 1.
+            newState.specifics.activeFrozenCores += 1
+            break
+          case 'W16':
+            newState.specifics.liveDawnsparks = true
+            break
+          case 'W2':
+            // Frosthexers
+            newState.specifics.potentialFrozenEnemiesInAColumn += 2
+            break
+          case 'W6':
+            // Moment's Peace
+            newState.specifics.potentialFrozenEnemiesInAColumn += 3
+            break
+          case 'W11':
+            // Midwinter Chaos
+            newState.specifics.potentialFrozenEnemiesInAColumn += 4
+            break
```

Comment:
I’m not sure how much value there is in counting enemies. Maybe we could simplify it like this:

```js
case 'W2':
case 'W6':
case 'W11':
  newState.specifics.potentiallyFrozenUnits = true
  break
```

Reply:
I was planning to implement Zhevana's ability after - that's why I'm only counting in a column